piksi_multi_cpp/include/piksi_multi_cpp/device.h

```diff
@@ -18,6 +16,8 @@ inline bool identifierEqual(const Identifier& a, const Identifier& b) {
 namespace piksi_multi_cpp {
 class Device {
  public:
+  enum Type { kUSB = 0 };
```

Comment:
Type has too much other meanings - this could be very ambigious - I would keep DeviceType